src/clusterfuzz/_internal/bot/tasks/utasks/minimize_task.py

```diff
@@ -353,43 +354,37 @@ def execute_parallel_runs(self, runs, instances=None):
 
 
 def utask_preprocess(testcase_id, job_type, uworker_env):
+  """Preprocess in a trusted bot."""
+  # Locate the testcase associated with the id.
+  testcase = data_handler.get_testcase_by_id(testcase_id)
+  if not testcase:
+    return None
+
   return uworker_io.UworkerInput(
       job_type=job_type,
-      testcase_id=str(testcase_id),
-      uworker_env=uworker_env,
-  )
-
-
-def utask_postprocess(output):
-  del output
+      testcase_id=testcase_id,
```

Comment:
I think testcase_id as given to preprocess will always be a string. But since it's stored as an int in the db, I prefer to cast to string explicitly.

Reply:
got it! done :D